docs/release-notes/2.2.0.asciidoc

```diff
@@ -0,0 +1,61 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-2.2.0]]
+== {n} version 2.2.0
+
+
+
+
+[[enhancement-2.2.0]]
+[float]
+=== Enhancements
+
```

Comment:
I am missing the biggest feature here: external secrets for associations!

Reply:
:smile:

Reply:
74a7e5a